lib/seattleflu/id3c/cli/command/clinical.py

```diff
@@ -996,6 +1045,14 @@ def parse_kp2023(kp2023_filename: str) -> None:
         'patient_class'
     ]
 
+    # throw a warning if there are any columns not in columns_to_keep
+    # except for known extra columns, like 'individual' which might not be included anymore but has been included in the past
+    extra_cols = list(set(clinical_records.columns).difference(columns_to_keep))
+    if 'individual' in extra_cols:
+        extra_cols.remove('individual')
+    LOG.warning(f'One or more unexpected columns present after parsing: {*extra_cols,}.\n\
+        Removing these columns before dumping records to stdout.')
```

Comment:
I think here you meant to only throw this warning if there were extra cols. At the moment, it will log even if the list is empty.
```suggestion
    if len(extra_cols) > 0:
        LOG.warning(f'One or more unexpected columns present after parsing: {*extra_cols,}.\n\
        Removing these columns before dumping records to stdout.')
```